### DISCUSSION TITLE
Propose new application to add

### DISCUSSION THREAD

**Discussion by OlivierGaland** [OlivierGaland/CashFactory#3] [General]
Topic opened for adding new apps in this package.

Any proposal will be studied, but keep in mind that : 
- Currently the project is focused on x86/amd64 architecture running linux with docker integration
- Application should be dockerizable (but it does not mean it provide a docker image, this can be developped by a tier)
- Application should be a passive income app using low memory,cpu,gpu and disk space : best candidates are bandwidth share apps.

**Comment by nittey** [comment]
Hi Olivier, Proxyrack finally has a Docker version. Can you add this to your stack? Look at this URL: https://hub.docker.com/r/proxyrack/pop

**Reply by OlivierGaland** [reply]
Hello, and many thanks for the advice.

I've created a new release (v1.4) that implement proxyrack application as a container.

**Comment by RocketDevil** [comment]
**New App "Spide Network"**  _and no its not Spider since its dead for good :D)_

ofc u get 5 bucks when u [**register via my ref**](https://spide.network/register.html?b7a62d), otherwise if u do without ref there system notices that and u get 0$ as starting bonus :D
its always nice to get another 5 bucks for free :D 👍 

- Linux Download: https://config-alpha-01.sgp1.digitaloceanspaces.com/Spide_linux64.zip
- Linux CLI Download:  https://config-alpha-01.sgp1.digitaloceanspaces.com/spide_linux_cli.zip
- Windows Download: https://config-alpha-01.sgp1.digitaloceanspaces.com/Spide.zip

**Reply by RocketDevil** [reply]
sadly no direct docker, but u shod be able to compose a rudimentary version of it into a container i guess with your skills ;)
